Doc 851 log-collector script instructions (#1486)
RedisLabs/redislabs-docs: content/platforms/kubernetes/tasks/run-log-collector-script.md

---
Title: Run the log collector script

> linkTitle: Run log_collector.py

Reviewer: ```suggestion
linkTitle: Run log_collector.py
description: Run the log collector script to package relevant logs into a tar.gz file to send to Redis Support for help troubleshooting your Kubernetes environment.
```

---
Title: Run the log collector script
linkTitle: Run log_collector.py
description: Run the log collector script to package relevant logs into a tar.gz file to send to Redis Support for help troubleshooting your Kubernetes environment.
weight: 89
alwaysopen: false
categories: ["Platforms"]
aliases: 
---

> The Redis Enterprise cluster (REC) log collector script ([`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py)) creates and fills a directory with the relevant logs for your environment.

Author: ```suggestion
The Redis Enterprise cluster (REC) log collector script ([`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py)) creates and fills a directory with the relevant logs for your environment. These logs will help the support team with troubleshooting. 
```

1. Download the [`log_collector.py`](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/log_collector/log_collector.py) file.
2. Run the script on the system that runs your `kubectl` or `oc` commands.

>     - Pass `-n` parameter to run on the current namespace.

Author: ```suggestion
    - Pass `-n` parameter to run on a different namespace than the one you are currently on.
```
